app/javascript/components/student/editor/TestList.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+export function TestsList({ tests }: { tests: Test[] }) {
+  return (
+    <div>
+      {tests.map((test: Test) => (
+        <p key={test.name}>
```

Comment:
Are names guaranteed to be unique?

Reply:
Within this array, yes. So if `key` is scoped to this block, this should be ok.